scripts/build-shrink-ray.js

```diff
@@ -29,16 +29,19 @@ if (sha) {
 }
 
 function buildPackages() {
-  execSync('yarn run storybook:build', {
+  execSync('yarn run storybook:build --quiet', {
     stdio: 'inherit',
   });
 }
 
 function setupShrinkRay() {
   console.log('[shrink-ray] Running shrink-ray prebuild script...');
 
-  // fetch latest in BuildKite pipeline
-  execSync('git fetch origin master');
+  // fetch latest in pipeline. Travis does a shallow clone by default,
+  // --unshallow makes sure we can fetch the master commit
+  execSync('git fetch --unshallow origin master:refs/remotes/origin/master', {
```

Comment:
why unshallow?

Reply:
Check the comment I ninjaed in just after this comment

Because travis does a shallow clone there's a chance that the commit at the tip of the master branch isn't included in the cloned commits. --unshallow grabs the full history to make sure we can find that commit when we do the rev-parse bits a few lines down